docs/review.md

```diff
@@ -0,0 +1,66 @@
+# Review Guidelines
+
+The following summarises review guidelines that we follow for pull requests in
+Riot Web and other supporting repos. These are just guidelines (not strict
+rules) and may be updated over time.
+
+## Code Review
+
+When reviewing code, here are some things we look for and also things we avoid:
+
+### We review for
+
+* Correctness
+* Performance
+* Accessibility
+* Security
+* Comments and documentation where needed
+* Sharing knowledge of different areas among the team
+* Ensuring it's something we're comfortable maintaining for the long term
+
+### We should avoid
+
+* Style nits that are already handled by the linter
+* Dramatically increasing scope
+
+### Good practices
```

Comment:
Should we include something about removing team review on larger PRs?

It hasn't been a problem in recent memory - just wondering if we should pick up the habit again.

Reply:
Good idea! I added some notes on review requests in the Workflow section.